### DISCUSSION TITLE
Gridsquare map - enhancements

### DISCUSSION THREAD

**Discussion by AndreasK79** [magicbug/Cloudlog#1123] [Show and tell]
I've been playing with the gridsquare map. I always wanted all the grids to show at a lower zoom level. After looking into the code, I figured it out. The problem is that the map becomes so slow when moving around, that it is not usable at all. As a compromise, I've done the following:

Start at zoom level 2 instead of 3. This is to show worked/confirmed fields.

Zoom level 2: (this is the same as before)
![image](https://user-images.githubusercontent.com/6977712/128606359-7313ccb0-8cb8-4233-b322-8ad86a6357f7.png)

Zoom level 3: Show all the field name labels and lines, and also show worked/confirmed gridsquares, but no labels (will be way too small to be readable).
![image](https://user-images.githubusercontent.com/6977712/128606392-2233e648-7234-4c88-b139-bcf945b8b2f2.png)

Zoom level 4: The same as zoom level 3.
![image](https://user-images.githubusercontent.com/6977712/128606403-9f4101ce-7c5f-4017-a5bd-e7e264274c9c.png)

Zoom level 5: Show only worked/confirmed gridsquares, nothing else. (Too slow if everything is shown)
![image](https://user-images.githubusercontent.com/6977712/128606440-91093576-04b3-4ae6-ba21-788b1823d9d6.png)

Zoom level 6:  and higher: Same as before.
![image](https://user-images.githubusercontent.com/6977712/128606727-cbdece52-7b52-40f4-b113-0e5cf7fcf3d7.png)

The map is modified to be clickable from zoom level 3 now (popup to show QSOs).

Are anyone interested in these enhancements?

**Comment by magicbug** [comment]
Yes this seems a good improvement imho

**Reply by AndreasK79** [reply]
Ok, I'll clean it up and submit a PR.
